Library/OAuth.swift

```diff
@@ -97,15 +97,15 @@ public struct OAuth {
     let params = OAuthTokenExchangeParams(temporaryToken: code, codeVerifier: verifier)
 
     AppEnvironment.current.apiService.exchangeTokenForOAuthToken(params: params)
-      .receive(on: RunLoop.main)
       .flatMap { response in
         let token = response.token
         // TODO: This would be neater if we can just return the V1 user from the exchange endpoint.
 
         // Return a publisher that emits a tuple of (token, user) when the user request completes
         return Just(token).setFailureType(to: ErrorEnvelope.self)
-          .zip(AppEnvironment.current.apiService.fetchUserSelf_combine(withToken: token))
+          .zip(AppEnvironment.current.apiService.fetchUserSelf_combine(withOAuthToken: token))
       }
+      .receive(on: RunLoop.main)
```

Comment:
Whoops, this wasn't applying to the `fetchUserSelf_combine` call because it happened before. 🤦‍♀️